src/styles/blocks/catalog.scss

```diff
@@ -1,3 +1,8 @@
+$card-width: 200px;
```

Comment:
Consider moving the variables to a separate file (e.g., _variables.scss) and import them, to make the code more maintainable and organized.